ir/src/main/scala/org/scalajs/core/ir/Definitions.scala

```diff
@@ -68,6 +68,10 @@ object Definitions {
 
   val ExportedConstructorsName = "__exportedInits"
 
+  val ReservedTags = Map("V" -> 0, "Z" -> 1, "C" -> 2, "B" -> 3,
+    "S" -> 4, "I" -> 5, "J" -> 6, "F" -> 7, "D" -> 8,
+    ObjectClass -> 9, CloneableClass -> 10, SerializableClass -> 11)
```

Comment:
This is an `Emitter`-specific implementation detail. It should be somewhere in `emitter` and be `private[emitter]`.